AUTHORS

```diff
@@ -0,0 +1,11 @@
+Aleksey Nishev <anishev@vmware.com>
+Andrew Karpow <andrew.karpow@sap.com>
+Andrew Karpow <andy@ndyk.de>
+Csaba Seres <45421502+Scsabiii@users.noreply.github.com>
+Johannes Kulik <43781403+joker-official@users.noreply.github.com>
```

Comment:
noreply.github.com mail addresses don't make much sense. Can we remove them?

Reply:
Yes, the AUTHORS was generated by the tox. I will remove them manually.